### PR TITLE
Fixes in accommodation and meeting registration workflow

### PR DESCRIPTION
**Pull request by pferreir**

### CODE REVIEW

File: indico/htdocs/js/indico/modules/registration/form/field.js [hunk 1]
```diff
@@ -869,7 +869,7 @@ ndRegForm.directive('ndAccommodationField', function(url) {
                 updateAccommodationPostData('departureDate', moment(newValue).format('YYYY-MM-DD'));
             });
 
-            scope.$watch('accommodation.typeId', function(newValue) {
+            scope.$watch('accommodation.choice', function(newValue) {
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/registration/form/field.js]: Really curious what happens the first time someone puts two accommodation fields in a form ;)

File: indico/htdocs/sass/base/_pages.scss [hunk 2]
```diff
@@ -61,3 +61,18 @@
...
+    h1 {
+        color: $dark-blue;
+    }
+}
+
+.meeting-page {
+    @extend %indico-page;
+    max-width: 800px;
+    margin: 2em auto 0 auto;
+}
```
Comment (ThiefMaster) [inline indico/htdocs/sass/base/_pages.scss]: missing trailing linebreak

File: indico/modules/events/registration/__init__.py [hunk 3]
```diff
@@ -168,3 +168,15 @@ class RegistrationRole(ManagementRole):
...
     description = _('Grants management access to the registration form.')
+
+
+@template_hook('event-header')
+def _inject_event_header(event, **kwargs):
+    from indico.modules.events.registration.models.forms import RegistrationForm
+
+    event = event.as_event
+    regforms = (event.registration_forms.filter_by(is_deleted=False, is_open=True)
+                     .order_by(db.func.lower(RegistrationForm.title)).all())
```
Comment (ThiefMaster) [inline indico/modules/events/registration/__init__.py]: usually we wrap sql queries like this:

```
regforms = (event.registration_forms
            .filter_by(is_deleted=False, is_open=True)
            .order_by(db.func.lower(RegistrationForm.title))
            .all())
```

more readable IMO (one line per chain element)

File: indico/modules/events/registration/templates/display/event_header.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                           {% set registration = session.user and regform.get_registration(user=session.user) %}
+                           {% if registration  %}
+                                <a class="i-button i-button-small icon-edit highlight"
+                                    href="{{ url_for('event_registration.display_regform', regform) }}">
+                                    <strong>{% trans %}Modify registration{% endtrans %}</strong>
+                                </a>
+                           {% elif regform.limit_reached %}
+                               <a class="i-button i-button-small highlight disabled"
+                                  title="{% trans %}There are no more places available.{% endtrans %}"
+                                  href="#">
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/event_header.html]: if you remove the `href` you don't get an ugly `#` in the url when clicking anyway

File: indico/modules/events/registration/templates/display/event_header.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                                {{ regform.title }}
+                            {% endif %}
+                        </span>
+                    </div>
+                    <div class="event-service-toolbar toolbar right">
+                        <div class="group">
+                           {% set registration = session.user and regform.get_registration(user=session.user) %}
+                           {% if registration  %}
+                                <a class="i-button i-button-small icon-edit highlight"
+                                    href="{{ url_for('event_registration.display_regform', regform) }}">
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/event_header.html]: `regform.locator.registrant`

File: indico/modules/events/registration/templates/display/regform_display.html [hunk 6]
```diff
@@ -10,7 +12,7 @@
 
 
 {% block subtitle %}
-    {% trans title=regform.title %}Add registration to "{{ title }}"{% endtrans %}
```
Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/regform_display.html]: `if management` - in management mode this is the correct subtitle

File: indico/htdocs/js/indico/modules/registration/form/field.js (hunk 1 above)
Comment (mic4ael) [inline indico/htdocs/js/indico/modules/registration/form/field.js]: If I am not mistaken, every field has its own scopem so it should not make any problems.

Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/registration/form/field.js]: Ah, I somehow thought it's `userdata`... anyway, we also do things with `userdata.accommodation`..

File: indico/modules/events/registration/templates/display/event_header.html (hunk 5 above)
Comment (pferreir) [inline indico/modules/events/registration/templates/display/event_header.html]: Fails miserably: 

```BuildError: ('event_registration.display_regform', {}, None)```

Comment (ThiefMaster) [inline indico/modules/events/registration/templates/display/event_header.html]: err, `registration.locator.registrant`

Comment (pferreir) [inline indico/modules/events/registration/templates/display/event_header.html]: Ah crap.

**Pull request closed without merging**